packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -674,3 +674,32 @@ const envoyUser = new iam.User(stack, 'envoyUser');
  */
 gateway.grantStreamAggregatedResources(envoyUser)
 ``` 
+
+## Adding Resources to shared meshes
+
+A shared mesh allows resources created by different accounts to communicate with each other in the same mesh.
+
+For more information about mesh sharing, see
+[Working with shared meshes](https://docs.aws.amazon.com/app-mesh/latest/userguide/sharing.html).
+
+
+In account A:
+
+```ts
+const mesh = new Mesh(stack, 'AppMesh', {
+  meshName: 'mySharedMesh',
+});
```

Comment:
Like I explained in https://github.com/aws/aws-cdk/pull/15353#discussion_r661625364, please roll back these changes, so that they only show an imported Mesh case.